src/reports/components/formatted-date.tsx

```diff
@@ -50,7 +50,7 @@ export class FormattedDate extends React.Component<FormattedDateProps> {
 
     private getTimeZone(date: Date): string {
         const timeString = date
-            .toLocaleTimeString('en-us', { timeZoneName: 'short' })
+            .toLocaleTimeString(undefined, { timeZoneName: 'short' })
```

Comment:
Even with this change, I think this datetime formatting logic is a little suspect; it's almost-but-not-quite ISO8601 (it uses 12h instead of 24h). We already have a runtime dependency on `moment`, so I'm wondering if it wouldn't be both simpler and more correct to replace all of `formatDateTime` and its helpers with something like

```ts
const keepTimeZoneOffset = true;
const momentDate = moment(this.props.date);
return momentDate.toISOString(keepTimeZoneOffset);
```